### ISSUE TITLE
Stopping Nicehash Crashes My Display Settings Vega 64

### ISSUE THREAD

**Issue by stevo10000** [nicehash/NiceHashMiner-Archived#1337]
Hi I have a rx vega 64 and whenever I press stop on nicehash all my wattman settings crash and radeon settings wont come back up, have to restart computer to open settings again. Just got Cryptonight (sgminer) to stay at 1.3 kh and now this. Settings also crash when algos switch sometimes but the settings always crash when stop is pressed. I have the latest 17.11 radeon drivers (I have tried the blockchain drivers and my computer crashes totally when even attempting to mine with them). I have tried to use basic settings without OC and UV but still it crashes every time on stop. Help

**Comment by bradtrebilcock** [comment]
I have the same issue with the vega 64. Running miner 1.8.1.5 on win10. Mine doesn't crash, but something significant happens for sure when stopping. I can start up again without rebooting. I also noticed these cards do not like the mining drivers. Works well with the latest gaming driver though as you said. 

⁣Sent from Blue ​

**Comment by stevo10000** [comment]
Are you having problems with setings getting wiped when algos switch at all? I think Im having that problem as well or the display drivers crash over time on the same algo, I leave the miner on for the night and check the next day and the settings have reverted back to normal settings so guessing it happened when the algos switched or after a long time on the same one>?

**Comment by bradtrebilcock** [comment]
Yes I have seen that too,  I was using afterburner and when it would do it my fan would reset and it would get hot. I am using wattman now and that doesn't happen with the fan but other settings revert. Mostly the power saving settings :(

⁣Sent from Blue ​

**Comment by stevo10000** [comment]
Found this closed https://github.com/nicehash/NiceHashMiner/issues/362 thread not sure why it was closed, could be we are using the cards as our dedicated video thats why we get the settings crash problem. I have some cap over my vga port, got to see if it works or not. Do you have the video going through the card also Brad?

**Comment by bradtrebilcock** [comment]
I experienced the problem when it was a secondary card also, no video hooked to it. That would make more sense in those cases though.

⁣Sent from Blue ​

**Comment by stevo10000** [comment]
Oh great I was hoping that would fix it, I found this thread https://github.com/nicehash/NiceHashMiner/issues/880 not sure if we can use this fix or not? Maybe we should message that dude that posted the fix cause he has yet to say anything on this? I only checked cryptonight (sgminer) and unchecked all the other so I dont loose my settings. Hopefully another alg. doesnt make much more or i'm missing out. :(

**Comment by stevo10000** [comment]
Finally got the blockchain drivers installed and they made a world of difference. Had to enable hbcc then disable it for nicehash not to crash my pc, also the drivers arent crashing when pressing stop. :)

**Comment by stevo10000** [comment]
Ok the only way to get the higher speeds with the blockchain drivers for me is I had to go to device manager and disable the vega then re enable it, and then I would get 1.9 kh on cryptonigh sgminer, if I didnt do that I would get only 1.3 to 1.5 (I have to do this before I start mining everytime). I think Ive been having these problems because of my old motherboard. I used these settings to get the most PROFIT a day and most STABLE speeds on wattman: -10% Core Freq, Memory 1025, Temp target 68, 0% power, and fan at 2000 min and 3900 target. Hope this helps people that were having problems like me. :) (Ill update this when I get new motherboard and let you know if I am having same problems).
